crates/neon-runtime/src/napi/tag.rs

```diff
@@ -35,7 +35,14 @@ pub unsafe extern "C" fn is_object(env: Env, val: Local) -> bool {
     is_type(env, val, napi::napi_valuetype::napi_object)
 }
 
-pub unsafe extern "C" fn is_array(_env: Env, _val: Local) -> bool { unimplemented!() }
+pub unsafe extern "C" fn is_array(env: Env, val: Local) -> bool {
+    let mut result = false;
+    if napi::napi_is_array(env, val, &mut result as *mut _) == napi::napi_status::napi_ok {
+        result
+    } else {
+        false
```

Comment:
Should this be asserting instead? It seems like it could be confusing if we say an object isn't an array when it is because the method failed for some other reason (e.g., throwing).

Reply:
think I agree — the other methods in this file do it this way but they should prob also panic.